tox.ini

```diff
@@ -29,9 +29,13 @@ deps =
     py{27}: enum34
 
     # Python3.5+ only deps
-    py{35,36,37,38}: aiohttp >= 3.0.0
+    py{35,36,37,38}: aiohttp >= 3.3.1
     py{35,36,37,38}: pytest-aiohttp
-    py{35,36,37,38}: aiobotocore >= 0.10.0
+    py{35,36,37,38}: botocore >= 1.15.3,<1.15.16
```

Comment:
<1.15.16 means we will have to keep updating this version number in the case of future updates with botocore. I think botocore >= 1.15.3 should be good.

Reply:
You're right. Probably don't need this one as well.